src/Function/Construct/IsEquivalence.agda

```diff
@@ -0,0 +1,63 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Some functional properties are Equivalence Relations
+--   This file is meant to be imported qualified.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Function.Construct.IsEquivalence where
+
+open import Data.Product using (_,_; swap)
+open import Function
+open import Level using (Level; _⊔_; suc)
+open import Relation.Binary hiding (_⇔_)
+open import Relation.Binary.PropositionalEquality using (setoid)
+
+import Function.Construct.Identity as I
```

Comment:
Any chance we could expand these to `Identity`, `Symmetry` and `Composition`? We're not exactly short on space here and it would make it that little bit more readable?

Reply:
Since you ask so nicely...